README.md

```diff
@@ -75,6 +76,27 @@ For example, to run QCEW numbers for one quarter:
 
     make -- run us.bls.QCEW --year 2014 --qtr 4
 
+If you want to use the local scheduler, you can add `SCHEDULER=--local-scheduler` to the `make` task
+
+### Execution watcher
+
+We've added a new script to watch for containers execution and notify when it ends. Is located in `scripts/watch_containers.py`
+
+```
+usage: python3 watch_containers.py [-h] [--since SINCE]
+                                   [--pooling-time POOLING_TIME]
+                                   [--notification-channel {stdout,slack,logfile}]
+                                   name
+```
+
+Let me explain the options:
+
+* **name**: Name, complete or a part of it, of the container to watch
+* **since**: [optional] Date in the `YYYY-mm-dd HH:MM:SS` format since the script starts looking for docker containers to watch. Default value is the now()
```

Comment:
The `StartedAt` property of the container uses UTC time. We should default it to `utcnow()` (or something like `datetime.utcnow()-timedelta(hours=1)`)